### ISSUE TITLE
large kissmanga images

### ISSUE THREAD

**Issue by bobula** [Twoure/KissNetwork.bundle#34]
Large images in kissmanga do not render correctly. Examples as follows:

When viewing the actual website the image is clear with no rendering issues.
(http://kissmanga.com/Manga/Dalbic-Jogaksa/Chapter-072?id=327987#5)
![004_1478579787](https://cloud.githubusercontent.com/assets/23389860/20197144/891ab892-a763-11e6-81fb-daf65d8686ef.jpg)

When viewed in an app (android app) or in pc chrome browser window, large images (even at maximum zoom level) do not render text clearly.
![screenshot_20161110-162850](https://cloud.githubusercontent.com/assets/23389860/20197096/4d077174-a763-11e6-8de0-b7112367cafd.png)

**Comment by Twoure** [comment]
Yes I've noticed this before as well.  It may be a limitation for Plex Clients, or the channel might not be pulling the full image.

When I have time, I'll take another look and see if there is anything to be done.
